langchain/utilities/graphql.py

```diff
@@ -0,0 +1,57 @@
+import json
+from typing import Any, Dict, Optional
+
+from pydantic import BaseModel, Extra, root_validator
+
+
+class GraphQLAPIWrapper(BaseModel):
+    """Wrapper around GraphQL API.
+
+    To use, you should have the ``gql`` python package installed.
+    This wrapper will use the GraphQL API to conduct queries.
+    """
+
+    custom_headers: Optional[Dict[str, str]] = None
+    graphql_endpoint: str
+    gql_client: Any  #: :meta private:
+    gql_function: Any  #: :meta private:
```

Comment:
if this is meant to be a callable can we type it as Callable

Reply:
Callable[[str], str] it seems

Reply:
🍰👍